lib/features/allow_ptrace.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * This feature adjusts the AppArmor profile to allow ptrace within a container.
+ * All of the interesting action takes place in `lib/task.js`.
+ */
+
+export default class AllowPtrace {
```

Comment:
Wow, this is really highlighting the need for making the "features" betters in the worker. Sorry that you had to do this nasty hack.